config/default.json

```diff
@@ -0,0 +1,38 @@
+{
+    "window_size": [
+        1920,
+        1013
+    ],
+    "window_position": [
+        -48,
+        39
+    ],
+    "module_width": 250,
+    "space_between_modules": [
+        50,
+        50
+    ],
+    "background_color": [
+        216,
+        255,
+        217
+    ],
+    "arrow": {
+        "width": 2,
+        "arrowWidth": 6,
+        "arrowLen": 6,
+        "space": [
+            0,
+            10
+        ],
+        "color": "QtGui.QColor(77, 120, 204)",
+        "borderWidth": 0,
+        "borderColor": "QtGui.QColor(77, 120, 204)"
+    },
+    "style": "Default",
+    "theme": "bright",
+    "filenames": [
+        "C:/Users/jean/Desktop/francis/resources/data/out\\graph_27062021 15h22m54.iag"
```

Comment:
That's not really good absolute path like that, it may be better to use relative path in this case?

Reply:
Ok but the iag file can be saved anywhere on your computer or external device. Then don't you think the absolute make sense? 
iag files should be managed like any file (txt, csv...)

Reply:
The main issue for me is just that we have your path. If this file does not exist in the folder, there is no reason to leave this path on the default code.

Reply:
Ah ok I get it! 

But I didn't explain properly the use of default.json, maybe that's why you see this as an issue:
In fact this file is updated each time you close the app in order to restore the previous settings when you re-open it. 
**Normally "filenames" is empty** but the last time I closed the app, multiple iag sessions were open. 
Plus, when you open the app if it cant find the default iag file it is not loaded.

Reply:
Maybe we should have a videoconf to talk about the management of iag sessions ?